html/css/widgets-styles.css

```diff
@@ -0,0 +1,23 @@
+.foreign {
+    text-align: left ;
+    /*width: 300px;*/
```

Comment:
If this and the comment in line 17 are here for a reason, could you please write it inside the comments? Else, can you better remove it?